felix/usagerep/usagerep_test.go

```diff
@@ -190,7 +191,7 @@ var _ = Describe("UsageReporter with mocked URL and short interval", func() {
 					q := url.Query()
 					Expect(q).To(HaveLen(expectedNumberOfURLParams), "unexpected number of URL parameters")
 					Expect(q.Get("guid")).To(Equal("someguid2"))
-					Expect(q.Get("type")).To(Equal("openstack,k8s,kdd,typha,bpf"))
+					Expect(q.Get("type")).To(Equal("openstack,k8s,kdd,typha,bpf,nft"))
```

Comment:
I sorta think this should be:
```
Expect(q.Get("type")).To(Equal("openstack,k8s,kdd,typha,bpf"))
```
and
```
Expect(q.Get("type")).To(Equal("openstack,k8s,kdd,typha,nft"))
```

Reply:
I think it's actually right - we'll never actually see this report, but the test just uses a single configuration on line 173.  It's not actually possible, but I think it's fine for the UT.